cmd/migrate.go

```diff
@@ -41,6 +43,12 @@ func NewMigrateCommand() *cobra.Command {
 		viper.AddConfigPath(path)
 	}
 
+	err := viper.ReadInConfig()
+	if err == nil {
+		viper.SetDefault(datastoreEngineFlag, viper.Get(datastoreEngineConf))
```

Comment:
why are we setting the default after reading the config file?

Reply:
When reading the config file, the values are accessible as nested keys (ie. `datastore.uri`). In `runMigration` value is accessed as `viper.GetString(datastoreURIFlag)`, that resolves to `datastore-engine` (ie. with dash).